test/ERC20RootVault.test.ts

```diff
@@ -664,6 +612,112 @@ contract<ERC20RootVault, DeployOptions, CustomContext>(
             });
 
             describe("edge cases:", () => {
+                describe("when lpPriceD18 <= hwmsD18", () => {
+                    it("do not charge performance fees", async () => {
+                        await this.erc20RootVaultGovernance
+                            .connect(this.admin)
+                            .stageDelayedProtocolParams({
+                                managementFeeChargeDelay: 0,
+                                oracle: this.mellowOracle.address,
+                            } as DelayedProtocolParamsStruct);
+
+                        await sleep(this.governanceDelay);
+                        await this.erc20RootVaultGovernance
+                            .connect(this.admin)
+                            .commitDelayedProtocolParams();
+
+                        var signer = await addSigner(randomAddress());
+                        const amount = BigNumber.from("3000000000000000010001");
```

Comment:
in my opinion it will be easier to read BigNumber.from(10).pow(18).add(DEPOSIT_MIN_AMOUNT)